ui/widgets/nuxeo-operation-button.js

```diff
@@ -328,19 +328,16 @@ import '../actions/nuxeo-action-button-styles.js';
     }
 
     _isRunning(status) {
-      if (status && status['entity-type'] === 'bulkStatus') {
-        const { state } = status.value || status;
-        return state !== 'ABORTED' && state !== 'COMPLETED';
-      }
-      return status === 'RUNNING';
+      const state = this._hasBulkStatus(status) ? (status.value && status.value.state) || status.state : status;
+      return state === 'RUNNING';
     }
 
     _isAborted(status) {
-      if (status && status['entity-type'] === 'bulkStatus') {
+      if (this._hasBulkStatus(status)) {
```

Comment:
Can't we do a one-liner like we did on the `_isRunning`?

Reply:
Sure, the logic is a bit different, but let me refactor this one too.

Reply:
Ah sorry, they seemed very similar. I'm ok with either, whatever is easier to read!